Source/Views/AlertView.swift

```diff
@@ -72,6 +73,15 @@ class AlertView: AlertControllerView {
         }
     }
 
+    private func createBorders() {
+        if let borderColor = self.visualStyle.borderColor {
+            self.layer.borderColor = borderColor.cgColor
+        }
+        if let borderWidth = self.visualStyle.borderWidth {
+            self.layer.borderWidth = borderWidth
+        }
+    }
```

Comment:
I'd assume setting to `nil` means resetting to default, there has to be _a_ color after all. But probably worth double-checking just to be sure.